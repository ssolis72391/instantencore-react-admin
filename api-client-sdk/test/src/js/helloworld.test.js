import dotenv from "dotenv";
import ApiClient from "../../../src/js/main";

dotenv.config();
const apiClient = new ApiClient();

apiClient.setEndPoint("https://dpb-api.zach.instantencore.com/1")
//apiClient.setAPIKey(process.env.X_API_KEY);

describe("client helloworld", () => {
  it("should return data", async () => {
    const response = await apiClient.read("helloworld");

    expect(response).toMatchSnapshot();

    expect(response).toMatchSnapshot({
      id: expect.any(Number),
      value: "Hello world 1 from api"
    });
  });
});
