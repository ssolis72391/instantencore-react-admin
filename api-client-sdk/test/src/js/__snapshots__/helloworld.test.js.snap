// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`client helloworld should return data 1`] = `
Object {
  "id": 1,
  "value": "Hello world 1 from api",
}
`;

exports[`client helloworld should return data 2`] = `
Object {
  "id": Any<Number>,
  "value": "Hello world 1 from api",
}
`;
